rosidl_pycommon/rosidl_pycommon/__init__.py

```diff
@@ -54,6 +54,31 @@ def get_newest_modification_time(target_dependencies):
             newest_timestamp = ts
     return newest_timestamp
 
+def render_visibility_control_file(package_name, template_path, base_output_path):
+    """
+    Render a visibility control file from a template.
+
+    :param package_name: Name of the ROS package for which
+      to generate the file.
+    :param template_path: Path to template visibility control file.
+      May contain @PROJECT_NAME@ and @PROJECT_NAME_UPPER@ placeholders,
+      to be substituted by the package name, accordingly.
+    :param base_output_path: Base output path for the generated file.
+
+    :returns: Path to the generated visibility control file.
+    """
+    with open(template_path, 'r') as fd:
+        content = fd.read()
+
+    content = content.replace('@PROJECT_NAME@', package_name)
+    content = content.replace('@PROJECT_NAME_UPPER@', package_name.upper())
```

Comment:
@frneer I think that mixing CMake templates with EmPy templates was not and is not a good idea in the long run. Consider migrating this template to EmPy to match all other templates.